datadog/resource_datadog_user.go

```diff
@@ -54,6 +67,11 @@ func resourceDatadogUser() *schema.Resource {
 				Optional:   true,
 				Deprecated: "This parameter was removed from the API and has no effect",
 			},
+			"roles": {
+				Type:     schema.TypeList,
```

Comment:
For preventing multiple values and ignoring order
```suggestion
				Type:     schema.TypeSet,
```